code/_onclick/hud/hud.dm

```diff
@@ -8,6 +8,7 @@ var/datum/global_hud/global_hud = new()
 /datum/global_hud
 	var/obj/screen/druggy
 	var/obj/screen/blurry
+	var/obj/screen/blind

```

Comment:
I made the blindness overlay into a global_hud (instead of being a var/obj/screen for each mob)